src/styles/Navbar.css

```diff
@@ -30,10 +39,28 @@
   font-family: Poppins, sans-serif;
   color: #28183b;
   font-size: 1rem;
+  position: relative;
+}
+
+.nav-item a::before{
```

Comment:
I think we can remove this hover state because someone is working on it on #734 , and your issue was primarily to add the name besides.